app/src/main/java/space/taran/arkretouch/presentation/drawing/EditManager.kt

```diff
@@ -50,13 +52,16 @@ class EditManager {
     val redoPaths = Stack<DrawPath>()
 
     var backgroundImage = mutableStateOf<ImageBitmap?>(null)
-
+    private val _backgroundColor = mutableStateOf(Color.White)
+    val backgroundColor: State<Color> = _backgroundColor
     private val backgroundImage2 = mutableStateOf<ImageBitmap?>(null)
     private val originalBackgroundImage = mutableStateOf<ImageBitmap?>(null)
 
     val matrix = Matrix()
     val editMatrix = Matrix()
 
+    private val _resolution = mutableStateOf<Resolution?>(null)
+    val resolution: State<Resolution?> = _resolution
     var drawAreaSize = mutableStateOf(IntSize.Zero)
     val availableDrawAreaSize = mutableStateOf(IntSize.Zero)
     val originalDrawAreaSize: IntSize
```

Comment:
What is difference between `resolution` and `drawAreaSize`?

Reply:
```drawAreaSize``` holds the whole device size and user don't decide it while ```resolution``` holds the size which user decides, it may equal ```availableDrawAreaSize```. But when background image is provided ```resolution``` has no use

Reply:
> But when background image is provided `resolution` has no use

Shouldn't original size of the loaded image be used as `resolution` in this case?

Reply:
If it's related to #38 then we can solve it in separate PR